_build_cfg.yml

```diff
@@ -83,8 +83,8 @@ Topics:
 Name: Using OpenShift
 Dir: using_openshift
 Topics:
-  - Name: Overview
-    File: overview
+#  - Name: Overview
+#    File: overview
```

Comment:
Yes, I do think we could do without an Overview for using_openshift, but I'll leave it to @CowboysFan to make the final decision so I'm certainly not touching the Welcome page. :)